src/clr-angular/popover/dropdown/dropdown-menu.ts

```diff
@@ -15,17 +29,19 @@ import { POPOVER_HOST_ANCHOR } from '../common/popover-host-anchor.token';
     `,
   host: {
     '[class.dropdown-menu]': 'true',
+    '[attr.role]': '"menu"',
   },
 })
-export class ClrDropdownMenu extends AbstractPopover {
+export class ClrDropdownMenu extends AbstractPopover implements AfterContentInit, OnDestroy {
   constructor(
     injector: Injector,
     @Optional()
     @Inject(POPOVER_HOST_ANCHOR)
-    parentHost: ElementRef,
+    parentHost: ElementRef<HTMLElement>,
     @Optional()
     @SkipSelf()
-    nested: ClrDropdownMenu
+    nested: ClrDropdownMenu,
+    focusHandler: DropdownFocusHandler
```

Comment:
nit `private focusHandler: DropdownFocusHandler` so we don't have to manually assign it later.

Reply:
So if I do that, TS compiler complains:
```A 'super' call must be the first statement in the constructor when a class contains initialized properties or has parameter properties.
```
I don't want to move up the `super` call, because I think the warning should fire rather than the `super` call if parentHost is not there. Is it ok to leave this as is?